src/resources/js/views/month-mobile-events.js

```diff
@@ -69,12 +74,55 @@ tribe.events.views.monthMobileEvents = {};
 			if ( contentId ) {
 				$content = $container.find( '#' + contentId );
 				tribe.events.views.accordion.closeAccordion( $header, $content );
+				hasDays = true;
 			}
 
 			obj.closeMobileEvents( $header, $content );
 		} );
 	};
 
+	/**
+	 * Handles the user selection of a day in the Grid.
+	 *
+	 * @since TBD
+	 *
+	 * @param {bool} viewHasDays Whether the day selected by the user has days in it or not.
+	 *
+	 * @return {void} The method does not return a value and will hide and show message elements.
+	 */
+	obj.handleMobileDayClick = function( hasDays ) {
+		var $defaultNotices = $( obj.selectors.mobileEventsDefaultNotices );
+		var $dynamicNotices = $( obj.selectors.mobileEventsDynamicNotices );
+
+		if ( hasDays ) {
+			$defaultNotices.hide();
+			$dynamicNotices.hide();
+			return;
+		}
+
+		var dynamicNoticesList = $dynamicNotices.find( obj.selectors.messageHeaderListItem );
+		var visibleMessages = 0;
+
+		if ( dynamicNoticesList.length > 0 ) {
+			dynamicNoticesList.each( function( index, item ) {
+				if ( item.getAttribute( 'data-key' ) !== 'no-events-in-day' ) {
+					item.style.display = 'none';
+				} else {
+					item.style.display = '';
+					visibleMessages++;
+				}
```

Comment:
Same as the other areas inverting the conditional can remove the `!==` operator

```suggestion
				if ( item.getAttribute( 'data-key' ) === 'no-events-in-day' ) {
					item.style.display = '';
					visibleMessages++;				
				} else {
					item.style.display = 'none';
				}
```